### DISCUSSION TITLE
Feature request (adwaita): Fuzzy search in tab overview

### DISCUSSION THREAD

**Discussion by FantomeBeignet** [ghostty-org/ghostty#4070] [Feature Requests, Ideas] [answered]
The title says it all, I think being able to fuzzy search your tabs in the overview could be a good UX improvement.

**Comment by jcollie** [comment, marked as answer]
The tab overview is implemented strictly by GTK/libadwaita so Ghostty has limited ability to customize it. You _can_ search (using the magnifying glass in the upper left or by typing some letters) but Ghostty cannot modify the algorithm that is being used.

**Reply by FantomeBeignet** [reply]
Alright, I'm not very familiar with gtk/libadwaita so I thought you had more control on how the search works. Thanks for your answer.
